pixeltable/utils/image.py

```diff
@@ -2,6 +2,8 @@
 
 
 def default_format(img: PIL.Image.Image) -> str:
+    if img.format == 'PNG':
```

Comment:
I don't know if this is the best fix possible, but without it, PNG images would end up as jpeg on disk.

Reply:
That might be what we want. Until now Pixeltable `pxt.Image` instances have been treated as "formatless" bitmaps. Let's discuss